tests/test_wave_05.py

```diff
@@ -12,24 +13,24 @@ def test_get_goals_no_saved_goals(client):
     assert response_body == []
 
 
-@pytest.mark.skip(reason="No way to test this feature yet")
+# @pytest.mark.skip(reason="No way to test this feature yet")
 def test_get_goals_one_saved_goal(client, one_goal):
     # Act
-    response = client.get("/goals")
+    response = client.get("/goals/1")
     response_body = response.get_json()
 
     # Assert
     assert response.status_code == 200
     assert len(response_body) == 1
-    assert response_body == [
-        {
+    assert response_body == {
+        "goal": {
             "id": 1,
             "title": "Build a habit of going outside daily"
         }
-    ]
+    }
```

Comment:
Note that this changes which endpoint is supposed to be tested there from `GET /goals` to `GET /goals/1`. So now there is no happy path test of that first endpoint.

I verified that it still works, but be careful about changing tests like this just to get them to pass. In this case you have changed this test to cover the exact functionality covered by the next test, `test_get_goal`.